media-types.md

```diff
@@ -5,7 +5,7 @@
 * Type name: `application`
 * Subtype name: `corim-signed+cbor`
 * Required parameters: n/a
-* Optional parameters: n/a
+* Optional parameters: "profile" (CoRIM profile in string format.  OIDs MUST use the dotted-decimal notation.)
```

Comment:
Is this an Optional parameter or a mandatory parameter? If optional and someone leaves it, the protocol will fail I suppose?

Reply:
it can't be mandatory because it's a mirror image of the `corim.profile`, which is an optional field.

Reply:
Agree, we fall back to default CoRIM Profile!!